docs/source/admin/traffic_stats.rst

```diff
@@ -91,35 +91,9 @@ To easily create databases, retention policies, and continuous queries, run :pro
-In Traffic Portal the :menuselection:`Other --> Grafana` menu item can be configured to display Grafana graphs using InfluxDB data (when not configured, this menu item will not appear). In order for this to work correctly, you will need two things:
-
-#. A :term:`Parameter` with the graph URL (more information below)
-#. The graphs created in Grafana. See below for how to create some simple graphs in Grafana. These instructions assume that InfluxDB has been configured and that data has been written to it. If this is not true, you will not see any graphs.
-
-To create a graph in Grafana, you can follow these basic steps:
-
-#. Login to Grafana as an administrative user
-#. Click on :menuselection:`Data Sources --> Add New`
-#. Enter the necessary information to configure your data source
-#. Click on :menuselection:`Home --> New` at the bottom
-#. Click on :menuselection:`"Collapsed Menu Icon" Button --> Add Panel --> Graph`
-#. Where it says :guilabel:`No Title (click here)` click and choose edit
-#. Choose your data source at the bottom
-#. You can have Grafana help you create a query, or you can create your own.
-
-	.. code-block:: postgresql
-		:caption: Sample Query
-
-		SELECT sum(value)*1000 FROM "monthly"."bandwidth.cdn.1min" GROUP BY time(60s), cdn;
-
-#. Once you have the graph the way you want it, click the :guilabel:`Save Dashboard` button at the top
-#. You should now have a new saved graph
-
-In order for Traffic Portal users to see Grafana graphs, Grafana will need to allow anonymous access. Information on how to configure anonymous access can be found on the configuration page of the `Grafana Website  <http://docs.grafana.org/installation/configuration/#authanonymous>`_.
-
-Traffic Portal uses custom dashboards to display information about individual :term:`Delivery Services` or :term:`Cache Groups`. In order for the custom graphs to display correctly, the Javascript files in :atc-file:`traffic_stats/grafana/`  need to be in the :file:`/usr/share/grafana/public/dashboards/` directory on the Grafana server. If your Grafana server is the same as your Traffic Stats server the RPM install process will take care of putting the files in place. If your Grafana server is different from your Traffic Stats server, you will need to manually copy the files to the correct directory.
-
-.. seealso:: More information on custom scripted graphs can be found in the `scripted dashboards <http://docs.grafana.org/reference/scripting/>`_ section of the Grafana documentation.
+- Traffic Control using Scenes for building dynamic Grafana dashboards, the Scenes app is built under: :file:`/var/lib/grafana/plugins/trafficcontrol-scenes-app`
+- To see Traffic Control dashboards, go to: :file:`https://grafanaHost/a/trafficcontrol-scenes-app`
+.. seealso:: More information on Grafana Scenes can be found in `Grafana Scenes introduction <https://grafana.com/blog/2023/08/03/new-in-grafana-10-grafana-scenes-for-building-dynamic-dashboarding-experiences/>`_
```

Comment:
Grammar:

- Traffic Control us<del>ing</del><ins>es</ins> Scenes <del>for</del><ins>to</ins> build<del>ing</del> dynamic Grafana dashboards<del>,</del><ins>.</ins> <del>t</del><ins>T</ins>he Scenes app is built under<del>:</del> ...
- ... can be found in \`<ins>the </ins>Grafana Scenes introduction...

Also, regarding "the Scenes app is built under" - are you saying that in order to build the "Scene" I have to copy my source files into that directory? Or is it just supposed to be stored there on a Grafana host?

`https://grafanaHost/a/trafficcontrol-scenes-app` is not a `file`. Also, is that (relative) URL guaranteed to be the location of the Scene on whatever Grafana host and configuration I choose to use?

Should these two sentences really be separate unordered list items, or can it just be a single paragraph?

Reply:
Currently, I let the trafficcontrol-scenes into Grafana CiaB folder, it means this plugin will be built and run when running CiaB with Grafana.
But I'm thinking about let this plugin inside Traffic Stats, and will be built with Traffic Stats. Then the user can use the folder: if the server is the same with Grafana, Grafana can use this plugin to display dashboards. If the server is different, they can copy it.
What do you think? @ocket8888

Reply:
I don't really know anything about Grafana, but all of our dashboards (is that even the right term?) currently live under `traffic_stats/grafana`, so that would seem a sensible place to put it.

Reply:
I updated the code and the docs to avoid confusion, the workflow stays the same, Traffic Control Grafana Scenes is built along with Traffic Stats. The only difference is that now people can visit: https://grafanaHost/a/trafficcontrol-scenes-app to view the dashboards